src/ucp/core/ucp_worker.c

```diff
@@ -2097,6 +2093,9 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
     return status;
 }
 
+static unsigned ucp_worker_discard_uct_ep_progress(void *arg);
+static unsigned ucp_worker_discard_uct_ep_destroy_progress(void *arg);
```

Comment:
i meant move the functions, to avoid the need for these

Reply:
done